### PR TITLE
Ensure Layman username at first login

### LINKED ISSUE #2923: Wagtail: Layman request /current-user does not return "username"
# Bug

**Describe the bug**
After using the "Login to catalogue" button in Wagtail-integrated app, after a successful login is done, a modal window pops-up. The "close" link in the modal window won't close it.

**To Reproduce**
1. Go to https://hub4everybody.lesprojekt.cz/map-demo/
2. Click on 'Datasources' and log in to catalogue
3. Click on the "close" text in the modal dialog
4. See nothing is happening

**Expected behavior**
The modal dialog window closed.

**Screenshots**
image

**Additional context**
none

### PR DESCRIPTION
**Pull request by fzadrazil**
## Description

Add missing username reservation in Layman when logging in from the map.

## Related issues or pull requests

Closes #2923 

## Pull request type

- [X] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [X] No
- [ ] I am unsure (no worries, we'll find out)

## Checklist

- [X] I understand and agree that the changes in this PR will be licensed under the [MIT License]
- [X] I have followed the [guidelines for contributing](https://github.com/hslayers/hslayers-ng/blob/master/CONTRIBUTING.md)
- [X] The proposed change fits to the content of the [code of conduct](https://github.com/hslayers/hslayers-ng/blob/master/CODE_OF_CONDUCT.md)
- [X] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [X] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

**Comment by fzadrazil** [comment]
@jmacura The fix is deployed on hub4everybody. Works for me now. Please try with your username.

**Review by jmacura — APPROVED** [review]
I can confirm it's all right now!
image
